app/components/developers/banner_component.html.erb

```diff
@@ -0,0 +1 @@
+<%= render selected_banner if selected_banner %>
```

Comment:
```suggestion
<%= render selected_banner %>
```

I _think_ that should work. Because `#render?` will return `false` if we don't have a banner to render so this will never get called.